### PR TITLE
adding logs for gather scripts [WIP]

### PR DESCRIPTION
**Pull request by OmerBenHayun**

### CODE REVIEW

File: tools/gather_logs_docker.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+rm -rf ovn-control-plane
+rm -rf ovn-worker
+rm -rf ovn-worker2
+mkdir -p ovn-control-plane
+mkdir -p ovn-worker
+mkdir -p ovn-worker2
+docker cp ovn-control-plane:/var/log/openvswitch/tcpdump.pcap .
+docker cp ovn-control-plane:/var/log/openvswitch/tcpdump_logs.log .
+kubectl --kubeconfig=/home/alexey/admin.conf get pods --all-namespaces -o wide > kube-topology
```
Comment (roytman) [inline tools/gather_logs_docker.sh:9]: could you please remove references to alexey ;-)

File: tools/gather_logs_docker.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+for d in ovn-control-plane ovn-worker ovn-worker2
+do
+		    docker cp $d:/var/log/openvswitch/ovn-controller.log $d/.  
+			    docker cp $d:/var/log/openvswitch/ovs-vswitchd.log $d/.     
+				    docker cp $d:/var/log/openvswitch/ovsdb-server.log $d/.  
+					    docker cp $d:/var/log/ovn-kubernetes/ovn-k8s-cni-overlay.log $d/.
+						        docker cp $d:/var/log/ovn-kubernetes/ovnkube.log $d/.
+						done
```
Comment (roytman) [inline tools/gather_logs_docker.sh:17]: I'd suggest to try to copy these files from all nodes, for the case when system runs in the HA mode.
May be we can send warning/error messages to /dev/null

File: tools/gather_logs_docker.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+rm -rf ovn-control-plane
```
Comment (roytman) [inline tools/gather_logs_docker.sh:1]: pls, rename the file to getLogs.sh

File: tools/get_logs.sh [hunk 4]
```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env bash
```
Comment (roytman) [inline tools/get_logs.sh:1]: I'd call this file as getEnv.sh or something similar, you gather much more info than only logs.

File: tools/gather_logs_docker.sh (hunk 2 above)
Comment (OmerBenHayun) [reply tools/gather_logs_docker.sh:18]: in `HA` mode there are three nodes:
```bash
NAME                STATUS     ROLES                  AGE   VERSION
ovn-control-plane   NotReady   control-plane,master   54m   v1.20.0
ovn-worker          NotReady   master                 53m   v1.20.0
ovn-worker2         NotReady   master                 53m   v1.20.0
```
those parts are part of the following for loop :
```bash
for d in ovn-control-plane ovn-worker ovn-worker2
do
    docker cp $d:/var/log/openvswitch/ovn-controller.log $d/.  
    docker cp $d:/var/log/openvswitch/ovs-vswitchd.log $d/.         
    docker cp $d:/var/log/openvswitch/ovsdb-server.log $d/.  
    docker cp $d:/var/log/ovn-kubernetes/ovn-k8s-cni-overlay.log $d/.
    docker cp $d:/var/log/ovn-kubernetes/ovnkube.log $d/.
done     
```
so as far as I understand we already copying those files from all the nodes.

File: tools/gather_logs_docker.sh (hunk 3 above)
Comment (OmerBenHayun) [reply tools/gather_logs_docker.sh]: Done

File: tools/get_logs.sh (hunk 4 above)
Comment (OmerBenHayun) [reply tools/get_logs.sh]: Done

File: tools/gather_logs_docker.sh (hunk 1 above)
Comment (OmerBenHayun) [reply tools/gather_logs_docker.sh]: reference  removed

**Pull request merged**